electron/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es2017",
+    "target": "es2021",
```

Comment:
@devcatalin any reason we are on such an old target?

Reply:
No reason as far as I know. Since we use electron then I think we can target newer versions without problems